api/api.yaml

```diff
@@ -0,0 +1,17 @@
+swagger: '2.0'
+info:
+  title: Data Explorer Service
+  description: API Service that reads from Elasticsearch.
+  license:
+    name: BSD
+    url: 'http://opensource.org/licenses/BSD-3-Clause'
+  version: '0.0.1'
+paths:
+  /facets:
+    get:
+      description: Returns facets.
+      responses:
+        200:
+          description: Facets.
+          schema:
+            type: string
```

Comment:
nit: If you add a `tags` here it will swagger codegen will auto generate different controllers per tag instead of putting it all in default_controller. Ex: https://github.com/DataBiosphere/job-manager/blob/master/api/jobs.yaml#L26